### PR TITLE
add copy to clipboard feature

### PR DESCRIPTION
**Pull request by scottsteinbeck**

### CODE REVIEW

**Comment by scottsteinbeck** [comment]
Screen Shot 2022-04-29 at 10 05 43 PM

File: views/doc.cfm [hunk 1]
```diff
@@ -288,6 +288,9 @@
 									<button class="example-btn btn btn-default" data-name="#encodeForHTMLAttribute(LCase(data.name))#" data-index="#example_index#"><span class="glyphicon glyphicon-play-circle"></span>&nbsp; Run Code</button>
 								</div>
 							</cfif>
+								<div class="pull-right" style="margin-right:10px">
+									<button class="copy-btn btn btn-default" onclick="copyTextToClipboard('#encodeForHTML(ex.code)#')" ><span class="glyphicon glyphicon-duplicate"></span>&nbsp; Copy Code</button>
```
Comment (KamasamaK) [inline views/doc.cfm:292]: I feel like "glyphicon-copy" would be better here.

Comment (scottsteinbeck) [reply views/doc.cfm]: updated

**Comment by KamasamaK** [comment]
I am seeing some JavaScript errors on this because the string being passed to `copyTextToClipboard` is not being encoded for JavaScript, so a single quote or new line will break it.

**Comment by KamasamaK** [comment]
I am getting JavaScript errors whenever I try this because the string being passed to `copyTextToClipboard` is not being encoded for JavaScript, so a single quote or new line will break it.

@scottsteinbeck @pfreitag Can you confirm?

**Comment by KamasamaK** [comment]
Never mind. I think #1488 probably fixes this.

**Pull request merged**